Makefile

```diff
@@ -6,22 +6,40 @@ rp:
 clean:
 	rm -f rp
 
-client:
+client: client-go client-python
+
+client-go:
 	go generate ./...
 	rm -rf pkg/client
 	sha256sum swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/preview/2019-12-31-preview/redhatopenshift.json >.sha256sum
 	sudo docker run \
+		--net host \
```

Comment:
this is breaking my docker builds... might be community version of docker or smth.

Reply:
that sounds like a problem to fix on your box, not in this PR?